tests/functional/smoke/full-compaction.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (C) 2021-2022 Micron Technology, Inc. All rights reserved.
+
+#doc: simple transaction test on a KVDB kvs
+
+. common.subr
+
+trap cleanup EXIT
+kvdb_create
+
+kvs=$(kvs_create smoke-0)
+
+cmd kmt "$home" "$kvs" -b -i128m -l16 -s1 -t10 -j64
+cmd hse kvdb compact -f -t 1200 $home
+
+uncompacted_nodecnt=$(cn_metrics $home $kvs | grep "^n " | awk '{print $3}' | grep -v 1 || echo 0)
```

Comment:
Need to rework this.  If the pipeline fails the output will be 0 and the test will pass.  I will push up this change to look for the "t" record with 1 node and 1 kvset and use grep's exit status.
```
cn_metrics "$home" "$kvs" | grep -q -P "^t\s+1\s+1\b"
[[ $? == 0 ]] || err "Tree is not fully compacted"
```

Reply:
```
cn_metrics | awk '/^n/ { if ($3 > 1) exit 1; }'
```

Reply:
Hmm... I tested with 1.  Dang it.  Seems like it should be 2.

Reply:
I was looking for `t 1 1` in the summary row.  Maybe `t [0-9]+ 1` would be better.

Reply:
dang it again.  you are right.  my fix is wrong. :(